packages/ui/utils/music.ts

```
@@ -0,0 +1,310 @@
+
+  if (artist.releaseGroups && artist.releaseGroups.length > 0) {
+    items.push({
+      label: `${artist.releaseGroups.length} albums`,
+      testid: 'artist-album-count',
+    })
+  }
+
+  return items
+}
+
+export function formatArtistType(type: string = 'artist'): 'artist' | 'soloist' | 'group' {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)